<launch>
  <!-- <node pkg="joy" type="joy_node" name="joy_node" />

    <node pkg="wavnodes" type="WavComms" name="WavComms" >
    <param name="base_port" type="string" value="/dev/ttyUSB0" />
    <param name="pub_odom" type="bool" value="true" />
    <param name="pub_odom_tf" type="bool" value="true" />
    <param name="pub_base_imu" type="bool" value="false" />

    <param name="head_port" type="string" value="/dev/ttyUSB2" />
    <param name="pub_head_imu" type="bool" value="true" />
    <param name="pub_head_imu_tf" type="bool" value="true" />
    <param name="joy_control" type="bool" value="true" />
  </node> 

  <node pkg="wavnodes" type="MotionComms" name="MotionComms" >
    <param name="port" type="string" value="/dev/ttyUSB0" />
    <param name="pub_odom" type="bool" value="true" />
    <param name="pub_odom_tf" type="bool" value="true" />
    <param name="pub_imu" type="bool" value="true" />
    <param name="joy_control" type="bool" value="true" />
  </node> 

  <node pkg="wavnodes" type="HeadComms" name="HeadComms" >
    <param name="port" type="string" value="/dev/ttyUSB1" />
    <param name="pub_imu" type="bool" value="false" />
    <param name="pub_imu_tf" type="bool" value="true" />
  </node>  
 

 <include file="$(find rplidar_ros)/launch/rplidar.launch" >
  <arg name="port" value="/dev/ttyUSB1" />
  </include>  -->

   <include file="$(find openni_launch)/launch/openni.launch" >
  <arg name="depth_registration" value="true" />
  </include>
 
 <!-- <group ns="rtabmap">
   <node pkg="nodelet" type="nodelet" name="rgbd_sync" args="standalone rtabmap_ros/rgbd_sync" output="screen">
      <remap from="rgb/image"        to="/camera/rgb/image_rect_color"/>
      <remap from="depth/image"      to="/camera/depth_registered/image_raw"/>
      <remap from="rgb/camera_info"  to="/camera/rgb/camera_info"/>

      <remap from="rgbd_image"       to="rgbd_image"/> 
      
      <param name="compressed_rate"  type="double" value="5"/>
     
      <param name="approx_sync"       value="true"/> 
    </node>
    </group> 

   <node pkg="tf" type="static_transform_publisher" name="tf_broadcaster" args="0 0 0 0 0 -0.707107 0.707107 base_link base_imu_link 200" /> -->



  
</launch>
